.github/workflows/check-typos.yml

```diff
@@ -4,13 +4,13 @@ on: [pull_request]
 jobs:
   run:
     name: typos
-    runs-on: "ubuntu-20.04"
+    runs-on: "ubuntu-latest"
```

Comment:
Why does this one use latest instead of 22.04?

Reply:
The [upstream job](https://github.com/crate-ci/typos/blob/master/.github/workflows/test-action.yml) uses ubuntu-latest as well, so I thought it makes sense to align with that. It doesn’t really matter though.